exercises/practice/atbash-cipher/.docs/instructions.md

```diff
@@ -0,0 +1,27 @@
+# Instructions
+
+Create an implementation of the atbash cipher, an ancient encryption system created in the Middle East.
+
+The Atbash cipher is a simple substitution cipher that relies on transposing all the letters in the alphabet such that the resulting alphabet is backwards.
+The first letter is replaced with the last letter, the second with the second-last, and so on.
+
+An Atbash cipher for the Latin alphabet would be as follows:
+
+```text
+Plain:  abcdefghijklmnopqrstuvwxyz
+Cipher: zyxwvutsrqponmlkjihgfedcba
+```
+
+It is a very weak cipher because it only has one possible key, and it is a simple mono-alphabetic substitution cipher.
+However, this may not have been an issue in the cipher's time.
+
+Ciphertext is written out in groups of fixed length, the traditional group size being 5 letters, leaving numbers unchanged, and punctuation is excluded.
```

Comment:
This is a very weird definition of atbash cipher; it generally just ignores punctuation and doesn't group letters in any way.
I opened a topic on the official forum about this https://forum.exercism.org/t/atbash-cipher-test-cases-might-be-wrong/12959

Reply:
If you're proposing a change to the canonical test suite, I'd be mindful that that problem spec has been around for ten years and has been implemented across 60+ tracks. It'd take some time and work to first sync the changes to the tracks, and then we'll need to rerun all the completed solutions. All solutions would fail if we're no longer grouping strings in five letter chunks. So you'll get some pushback for sure.

Reply:
It's not that we can't change the test suite, but we should have a good reason like there's an edge case to be tested that might be significant for multiple tracks.

Reply:
It was just seemed confusing, because I know what the actual atbash cipher requires. That's why I opened a topic on the forum to discuss it, but this won't affect this PR, which will be reviewed with the current problem description in mind.